packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -48,11 +48,11 @@ const Color _kDisabledBackground = CupertinoDynamicColor.withBrightness(
   darkColor: Color(0xFF050505),
 );
 
-// Value inspected from Xcode 11 & iOS 13.0 Simulator.
+// Value inspected from Xcode 12 & iOS 14.0 Simulator.
 // Note it may not be consistent with https://developer.apple.com/design/resources/.
 const CupertinoDynamicColor _kClearButtonColor = CupertinoDynamicColor.withBrightness(
-  color: Color(0xFF636366),
-  darkColor: Color(0xFFAEAEB2),
+  color: Color(0xFFCCCCCC),
```

Comment:
This is "the fourth thing". Our colors are a bit off now.

Reply:
nit: on iOS 14 this seems to be pure black with 20% alpha (0x33000000) and pure white with 20% alpha (0x33FFFFF) for light and dark.